crawler.py

```diff
@@ -203,7 +210,15 @@ def run_all(driver):
 
     # Webdriver uses Firefox Binaries from downloaded cov build
     driver = webdriver.Firefox(firefox_binary='tools/firefox/firefox-bin')
+
+    # All steps are stored in data folder
     run_all(driver)
+    random_name_for_folder = ''.join(random.choice(string.ascii_uppercase + string.digits) for _ in range(10))
```

Comment:
To generate a random name, you can use the uuid module, something like: `str(uuid.uuid4())`.